src/ASM/ASMs3Drecovery.C

```diff
@@ -115,10 +115,12 @@ Go::SplineVolume* ASMs3D::projectSolution (const IntegrandBase& integrand) const
 {
   PROFILE2("ASMs3D::projectSolution");
 
+  const int basis = 1;
+
```

Comment:
Det samme gjelder vel for ASMs2Drecovery-metoden. Kunne ellers tenkt meg en temporary;

Go::SplineVolume* bvol = this->getBasis(basis);

og så bruke denne (bvol)  i stedet for å kalle getBasis mange ganger.